### ISSUE TITLE
Integration tests fail

### ISSUE THREAD

**Issue by sophokles73** [eclipse/hono#147]
When running the integration tests, the Hono server does not start up anymore.

I suspect this is is because of the recent changes in the configuration properties ...

```sh
HONO> 
HONO> .   ____          _            __ _ _
HONO> /\\ / ___'_ __ _ _(_)_ __  __ _ \ \ \ \
HONO> ( ( )\___ | '_ | '_| | '_ \/ _` | \ \ \ \
HONO> \\/  ___)| |_)| | | | | || (_| |  ) ) ) )
HONO> '  |____| .__|_| |_|_| |_\__, | / / / /
HONO> =========|_|==============|___/=/_/_/_/
HONO> :: Spring Boot ::        (v1.4.0.RELEASE)
HONO> 
HONO> 13:04:03.100 [main] INFO  o.e.hono.application.Application - Starting Application on 806e4a95b7f4 with PID 1 (/opt/hono/hono-app-0.5-M5-SNAPSHOT.jar started by hono in /home/hono)
HONO> 13:04:03.107 [main] INFO  o.e.hono.application.Application - The following profiles are active: registration-file,qpid,prod
HONO> 13:04:03.201 [main] INFO  o.s.c.a.AnnotationConfigApplicationContext - Refreshing org.springframework.context.annotation.AnnotationConfigApplicationContext@2aaf7cc2: startup date [Mon Apr 03 13:04:03 GMT 2017]; root of context hierarchy
HONO> 13:04:04.763 [main] WARN  org.eclipse.hono.server.HonoServer - Hono server does not yet support limiting the incoming message size via the maxPayloadSize property
HONO> 13:04:04.878 [main] INFO  o.e.hono.application.HonoApplication - Starting authentication service org.eclipse.hono.authentication.impl.AcceptAllPlainAuthenticationService@587d1d39
HONO> 13:04:04.899 [main] INFO  o.e.hono.application.HonoApplication - Starting authorization service org.eclipse.hono.authorization.impl.InMemoryAuthorizationService@42d8062c
HONO> 13:04:04.904 [main] INFO  o.e.hono.application.HonoApplication - Starting registration service FileBasedRegistrationService[filename=device-identities.json]
HONO> 13:04:04.915 [vert.x-eventloop-thread-0] INFO  o.e.h.a.i.BaseAuthenticationService - listening on event bus [address: authentication.in] for incoming authentication messages
HONO> 13:04:04.918 [vert.x-eventloop-thread-1] INFO  o.e.h.a.i.BaseAuthorizationService - listening on event bus [address: authorization.in] for incoming auth messages
HONO> 13:04:04.919 [vert.x-eventloop-thread-1] INFO  o.e.h.a.i.InMemoryAuthorizationService - loading permissions from resource file:/etc/hono/permissions.json
HONO> 13:04:04.927 [vert.x-eventloop-thread-0] INFO  o.e.h.r.i.FileBasedRegistrationService - listening on event bus [address: registration.in] for incoming registration messages
HONO> 13:04:04.929 [vert.x-eventloop-thread-0] INFO  o.e.h.r.i.FileBasedRegistrationService - saving device identities to file every 3 seconds
HONO> 13:04:05.003 [vert.x-eventloop-thread-3] WARN  org.eclipse.hono.server.HonoServer - AMQP 1.0 secure port number configured, but the certificate setup is not correct. No secure port will be opened - please check your configuration!
HONO> 13:04:05.020 [vert.x-eventloop-thread-3] ERROR o.e.hono.application.HonoApplication - cannot start up HonoServer
HONO> io.vertx.core.impl.NoStackTraceThrowable: cannot start up HonoServer - no listening ports found.
HONO> 13:04:05.032 [vert.x-eventloop-thread-0] INFO  o.e.h.r.i.FileBasedRegistrationService - unregistered registration data consumer from event bus
HONO> 13:04:05.124 [main] INFO  o.e.hono.application.HonoApplication - Hono server has been shut down successfully
HONO> 13:04:05.417 [main] INFO  o.s.j.e.a.AnnotationMBeanExporter - Registering beans for JMX exposure on startup
HONO> 13:04:05.438 [main] INFO  o.e.hono.application.Application - Started Application in 3.396 seconds (JVM running for 4.385)
HONO> 13:04:05.440 [Thread-3] INFO  o.s.c.a.AnnotationConfigApplicationContext - Closing org.springframework.context.annotation.AnnotationConfigApplicationContext@2aaf7cc2: startup date [Mon Apr 03 13:04:03 GMT 2017]; root of context hierarchy
HONO> 13:04:05.442 [Thread-3] INFO  o.s.j.e.a.AnnotationMBeanExporter - Unregistering JMX-exposed beans on shutdown
HONO> 13:04:05.443 [Thread-3] INFO  o.e.hono.application.HonoApplication - Hono server has been shut down successfully
```

**Comment by sysexcontrol** [comment]
Had a look at it - is a configuration issue in the integration tests caused by the decision to only allow insecure ports when explicitly enabled.
Will provide a fix quite soon!
Thanks for reporting...

**Comment by sophokles73** [comment]
fixed in a0ab27f647d828a0b947e0f149013be432f51d90
